tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/plugin.py

```diff
@@ -32,18 +32,17 @@
 from tensorboard import errors
 from tensorboard import plugin_util
 from tensorboard.backend import http_util
-from tensorboard.data import provider
 from tensorboard.plugins import base_plugin
-from tensorboard_plugin_example import metadata
 from tensorboard.util import tensor_util
+from tensorboard.plugins.scalar import metadata
 
-scalar_plugin_name = 'scalars'
-plugin_folder = '/data/plugin/example_raw_scalars/'
+_SCALAR_PLUGIN_NAME = metadata.PLUGIN_NAME
+_PLUGIN_DIRECTORY = '/data/plugin/example_raw_scalars/'
```

Comment:
How about `_PLUGIN_DIRECTORY_PATH_PART`? I was confused a bit that we are using an absolute path here.

Reply:
Done.